vercel/resource_deployment_model.go

```diff
@@ -187,5 +205,19 @@ func convertResponseToDeployment(response client.DeploymentResponse, plan Deploy
 		PathPrefix:      fillStringNull(plan.PathPrefix),
 		ProjectSettings: plan.ProjectSettings.fillNulls(),
 		DeleteOnDestroy: plan.DeleteOnDestroy,
+		GitSource:       plan.GitSource,
+	}
+}
+
+// checkMutualyExclusiveAttributes checks whether git_source and files are not set at the same time
+func (d *Deployment) checkMutualyExclusiveAttributes() error {
```

Comment:
I think we can do this as a validation instead, but I'll merge the changes and fix this up retrospectively.

Reply:
that makes sense

Reply:
just to clarify, are you merging this or you require more changes?